tests/performance/group_array_sorted.xml

```diff
@@ -1,27 +1,11 @@
 <test>
-    <settings>
-        <max_threads>10</max_threads>
-    </settings>
-    <substitutions>
-        <substitution>
-            <name>items</name>
-            <values>
-                <value>1000</value>
-                <value>100000</value>
-                <value>10000000</value>
-            </values>
-        </substitution>
-    </substitutions>
+    <create_query>CREATE TABLE test (`id` UInt64, `value` UInt64, `text` String ) ENGINE = Memory</create_query>
+    <fill_query>INSERT INTO test SELECT number as id, rand64() as value, toString(number) as text FROM system.numbers_mt LIMIT 10000000</fill_query>
+
+    <query>SELECT groupArraySorted(100000)(id, value) FROM test</query>
+    <query>SELECT groupArraySorted(100000)(text, value) FROM test</query>
+    <query>SELECT groupArraySorted(100000)((id, text), value) FROM test</query>
+    <query>SELECT groupArraySorted(100000)(text) FROM test</query>
 
```

Comment:
I think these were meant to test different limit sizes cc @palegre-tiny

Reply:
Yes, that was the idea. Testing performance for difference dataset sizes.